app/helpers/field_validation_helper.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 module FieldValidationHelper
-  def validate_mandatory_text_fields(mandatory_fields, page)
+  def validate_mandatory_text_fields(mandatory_fields, page, form_responses)
```

Comment:
Where is this being used? I was expecting to see where it's called needing to be updated too. 🤔

Reply:
Doesn't look like we are using it anywhere.  I'll remove it.